espnet2/layers/stft.py

```diff
@@ -115,7 +117,29 @@ def forward(
         return output, olens
 
     def inverse(
-        self, input: torch.Tensor, ilens: torch.Tensor = None
+        self, input: Union[torch.Tensor, ComplexTensor], ilens: torch.Tensor = None
     ) -> Tuple[torch.Tensor, Optional[torch.Tensor]]:
-        # TODO(kamo): torch audio?
-        raise NotImplementedError
+        """
+
+        :param input: Tensor (batch, T, F, 2) or ComplexTensor (batch, T, F)
+        :param ilens:
+        :return:
+        """
+        if isinstance(input, ComplexTensor):
+            input = torch.stack([input.real, input.imag], dim=-1)
+        assert input.shape[-1] == 2
+        input = input.transpose(1, 2)
+
+        wavs = torchaudio.functional.istft(
```

Comment:
FYI: I have moved `istft` implementation to `pytorch` (it's exactly the same implementation but in C++), and `torchaudio` will soon drop this.
I am not sure which version of PyTorch ESPNet2 uses, but if ESPNet2 is expecting users to use the latest PyTorch, then you will likely want to call it as `torch.functional.istft`.